cmake/ElektraCache.cmake

```diff
@@ -341,73 +263,62 @@ endif ()
 # Misc.
 #
 
-set (LIB_SUFFIX ""
-		CACHE STRING
-		"Optional suffix to use on lib folders (e.g. 64 for lib64)"
-    )
+set (LIB_SUFFIX "" CACHE STRING "Optional suffix to use on lib folders (e.g. 64 for lib64)")
 
-set (MEMORYCHECK_COMMAND_OPTIONS "--error-exitcode=1" )
+set (MEMORYCHECK_COMMAND_OPTIONS "--error-exitcode=1")
 set (MEMORYCHECK_SUPPRESSIONS_FILE
-		"${CMAKE_SOURCE_DIR}/tests/valgrind.suppression"
-		CACHE FILEPATH
-		"Full path to suppression file for valgrind")
+     "${CMAKE_SOURCE_DIR}/tests/valgrind.suppression"
+     CACHE FILEPATH
+	   "Full path to suppression file for valgrind")
 
-
-
-set(DISCLAMER "
+set (
+	DISCLAMER
+	"
 /***************************************************************************
  *                                                                         *
  * This header file is AUTO GENERATED.                                     *
  * Do not edit this file, otherwise your changes will be discarded in the  *
  * next cmake run.                                                         *
  *                                                                         *
- ***************************************************************************/")
-
-
-MARK_AS_ADVANCED(FORCE
-	# might be relevant to a few users:
-	GTEST_ROOT
```

Comment:
I think the manual groups are better than the reformatted source.

Reply:
I think so too. Unfortunately `cmake-format` does not support a method to disable formatting for a (non-comment) block as far as I know.